src/js/disability-benefits/526EZ/components/DisabilityWizard.jsx

```diff
@@ -0,0 +1,191 @@
+
+    this.state = {
+      currentLayout: chooseStatus
+    };
+  }
+
+    ButtonContainer = () => {
+      const { currentLayout, disabilityStatus, add, increase } = this.state;
+      const { profile, loginUrl, verifyUrl } = this.props;
+      const notUpdatingStatus = disabilityStatus === 'first' || disabilityStatus === 'appeal';
+      const updatingStatus = disabilityStatus === 'update';
+      const eligibleForIncrease = updatingStatus && !add && increase;
+      const ineligibleForIncrease = updatingStatus && add;
+      const atIncreaseGuidance = currentLayout === applyGuidance && eligibleForIncrease;
+      const atEbenefitsGuidance = currentLayout === applyGuidance && (notUpdatingStatus || ineligibleForIncrease);
+
+      return  (<div>
+        {currentLayout !== chooseStatus &&
+        <button type="button" className="usa-button-secondary" onClick={this.goBack}><span className="button-icon">« </span>Back</button>
+        }
+        {atIncreaseGuidance && !sessionStorage.userToken &&
+        <a className="usa-button-primary" href="/disability-benefits/526/apply-for-increase/introduction/" onClick={this.authenticate}>Sign in<span className="button-icon"> »</span></a>
+        }
+        {atIncreaseGuidance && sessionStorage.userToken &&
+        <RequiredLoginView
+          containerClass="login-container"
+          authRequired={1}
+          serviceRequired={['disability-benefits']}
+          userProfile={profile}
+          loginUrl={loginUrl}
+          verifyUrl={verifyUrl}>
+          <a className="usa-button-primary" href="/disability-benefits/526/apply-for-increase/introduction/">Apply now<span className="button-icon"> »</span></a>
+        </RequiredLoginView>}
+        {atEbenefitsGuidance &&
+        <a className="usa-button-primary" href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=disability-compensation">Go to eBenefits to Apply<span className="button-icon"> »</span></a>
+        }
+        {currentLayout !== applyGuidance &&
+        <a className="usa-button-primary" onClick={this.goForward}>Next<span className="button-icon"> »</span></a>
+        }
+      </div>);
+    }
+
+  answerQuestion = (field, answer) => {
+    const newState = Object.assign({}, { [field]: answer });
+    this.setState(newState);
+  };
+
+  goToNextPage = () => {
+    let nextLayout = applyGuidance;
+    if (this.state.currentLayout === chooseStatus && this.state.disabilityStatus === 'update') {
+      nextLayout = chooseUpdate;
+    }
+    this.setState({ currentLayout: nextLayout, errorMessage: '' });
+  }
+
+  goBack = () => {
+    let nextLayout = chooseStatus;
+    if (this.state.currentLayout === applyGuidance && this.state.disabilityStatus === 'update') {
+      nextLayout = chooseUpdate;
+    }
+    this.setState({ currentLayout: nextLayout, errorMessage: '' });
+  };
+
+  displayErrorMessage = () => {
+    this.setState({ errorMessage: 'Please select an option' });
+  }
+
+  GetStartedMessage = () => {
```

Comment:
Seems odd to have a property of a component be another component. What do you think of pulling this (and `ButtonContainer` above) out into separate components and passing the state from `DisabilityWizard` as props? Might also help for testing that way.

Reply:
Good idea, updated.